src/PublicApiGenerator/ApiGenerator.cs

```diff
@@ -290,7 +291,7 @@ private static CodeTypeDeclaration CreateTypeDeclaration(TypeDefinition publicTy
                 declaration.BaseTypes.Add(publicType.BaseType.CreateCodeTypeReference(publicType));
             }
         }
-        foreach (var @interface in publicType.Interfaces.OrderBy(i => i.InterfaceType.FullName, StringComparer.Ordinal)
+        foreach (var @interface in publicType.Interfaces.OrderBy(i => i.InterfaceType, typeComparer)
```

Comment:
I'm not sure about it. This sorts implemented interfaces like `A : I1, I2`. I think sorting should always work as `.OrderBy(i => i.InterfaceType.FullName, StringComparer.Ordinal)`.

Reply:
The same for generic parameters and type members. In other words new setting should work only for "top-level entities" - types/namespaces.